camera_manager:
  ros__parameters:
    sim: true
    camera_id: [1, 2]
    image_topic: [
      "/robostore/left_hand/color/image_raw", 
      "/robostore/right_hand/color/image_raw"
      ]
    pointcloud_topic: [
      "/robostore/left_hand/depth/color/points", 
      "/robostore/right_hand/depth/color/points"
      ]